compiler/common-artifacts/CMakeLists.txt

```diff
@@ -221,46 +221,46 @@ foreach(RECIPE IN ITEMS ${RECIPES})
 
   set(MODEL_FILE "${RECIPE}${OPT_FORMAT}.${MODEL_FORMAT}")
   set(MODEL_PATH "${CMAKE_CURRENT_BINARY_DIR}/${MODEL_FILE}")
-  set(NNPKG_FILE "${RECIPE}${OPT_FORMAT}")
+  set(NNPKG_FILE "${RECIPE}")
   set(NNPKG_PATH "${CMAKE_CURRENT_BINARY_DIR}/${NNPKG_FILE}")
 
   add_custom_command(OUTPUT ${NNPKG_PATH}
-    COMMAND ${MODEL2NNPKG} ${MODEL_PATH}
+    COMMAND ${MODEL2NNPKG} -p ${RECIPE} ${MODEL_PATH}
     DEPENDS ${MODEL2NNPKG} ${MODEL_PATH}
     COMMENT "Generate ${RECIPE} nnpackage"
   )
   list(APPEND TEST_DEPS ${NNPKG_PATH})
 
   set(INPUT_HDF5_FILE "${RECIPE}${OPT_FORMAT}.input.h5")
   set(INPUT_BIN_PATH "${CMAKE_CURRENT_BINARY_DIR}/${INPUT_HDF5_FILE}")
-  
+
   set(EXPECTED_HDF5_FILE "${RECIPE}${OPT_FORMAT}.expected.h5")
   set(EXPECTED_BIN_PATH "${CMAKE_CURRENT_BINARY_DIR}/${EXPECTED_HDF5_FILE}")
-  
+
   if(NOT DEFINED NO_TCGEN_${RECIPE})
     # Generate input.h5, expected.h5
     add_custom_command(OUTPUT ${INPUT_BIN_PATH} ${EXPECTED_BIN_PATH}
       COMMAND $<TARGET_FILE:testDataGenerator> ${MODEL_FILE}
       DEPENDS $<TARGET_FILE:testDataGenerator> ${MODEL_FILE}
       COMMENT "Generate ${INPUT_BIN_PATH} and ${EXPECTED_BIN_PATH}"
     )
-    
+
     # Generate test directory
     set(TC_DIRECTORY "${NNPKG_PATH}/metadata/tc")
     add_custom_command(OUTPUT ${TC_DIRECTORY}
       COMMAND ${CMAKE_COMMAND} -E make_directory ${TC_DIRECTORY}
       DEPENDS ${NNPKG_PATH}
       COMMENT "Generate ${RECIPE} nnpackage test directory"
     )
-    
+
```

Comment:
@mhs4670go or @hseok-oh , if this PR is just closed, could you add another PR for these remove space changes?

Reply:
I'll post a PR.
@hseok-oh Please close this PR:)